components/playback/controls/YAxisControls.vue

```diff
@@ -234,32 +236,36 @@ export default {
     },
     y_axis_controls_commit: function (new_range) {
       this.$bvModal.hide("y-axis-controls-settings");
-      const default_y_axis_zoom_idx = 0;
-      const new_y_zoom_levels = [
-        { y_min: new_range.y_min, y_max: new_range.y_max },
-        { y_min: 0, y_max: 200 },
-        { y_min: 25, y_max: 150 },
-        { y_min: 50, y_max: 150 },
-        { y_min: 50, y_max: 140 },
-        { y_min: 50, y_max: 130 },
-        { y_min: 55, y_max: 120 },
-        { y_min: 60, y_max: 120 },
-        { y_min: 60, y_max: 110 },
-        { y_min: 65, y_max: 110 },
-        { y_min: 70, y_max: 110 },
-        { y_min: 70, y_max: 105 },
-        { y_min: 75, y_max: 105 },
-        { y_min: 80, y_max: 105 },
-        { y_min: 85, y_max: 105 },
-        { y_min: 90, y_max: 105 },
-        { y_min: 94, y_max: 102 },
-        { y_min: 96, y_max: 100 },
-      ];
+      let duplicate_idx = null;
+
+      const new_element = { y_min: new_range.y_min, y_max: new_range.y_max };
 
-      this.$store.commit("waveform/set_y_axis_zoom_levels", new_y_zoom_levels);
+      this.y_axis_zoom_levels.splice(0, 0, new_element); // insert the new element
```

Comment:
Do you intend to leave every user entered value in the list?

Reply:
Don't understand your question, the new_element which user entered is put into the array is what would the execution do.

Reply:
if the user changes the custom zoom level won't this add another to the list and leave the old custom zoom level in the list also?

Reply:
Any Y-min/Y-max which is not present in the list if added will be inserted into the list. Its possible sometimes the user might enter the value which pre-exists in the list then its just identified as duplicate and removed, but the y_zoom_idx level will be updated to the value present in the list.